index.php

```diff
@@ -0,0 +1,29 @@
+<?php
+    include "includes/header.php";
+
+    //write PHP here...
+    $weight = '225';
+    $age = '43';
+    $first_name = 'Tony';
+    $last_name = 'Mainelli';
+
+    function poundsToKg()
+    {
+        global $weight, $Kg;
+
+        $Kg = $weight * (1/ 2.2);
+    }
```

Comment:
this will work - but the assignment asked for you to take in the weight in lbs as an arg. So, I expected something like:

function poundsToKG($w){
    return $w *(1/2.2);
}